File: Latihan A/cpp/Ram.cpp
// Library
#include <iostream>
#include <string>
using namespace std;
// deklarasi kelas
class Ram{
    // variabel private
    private:
        string capacity;
        int price;
    // variabel dan method publik
    public:
        // konstruktor
        Ram(){

        }
        Ram(string capacity, int price){
            this->capacity = capacity;
            this->price = price;
        }
        // get dan set
        void setcapacity(string capacity){
            this->capacity = capacity;
        }
        string getcapacity(){
            return this->capacity;
        }
        void setprice(int price){
            this->price = price;
        }
        // method tampilkan
        int getprice(){
            return this->price;
        }
        // destruktor
        ~Ram(){}
};

File: Latihan B/php/Ram.php
<?php
// deklarasi kelas
class Ram{
	//variabel lokal
	private $capacity;
    private $price;
	//konstruktor
	public function __construct($capacity = "", $price = 0){
		$this->price = $price;
        $this->capacity = $capacity;
	}
	//get dan set setiap variabel
	public function setprice($price){
		$this->price = $price;
	}
	public function getprice(){
		return $this->price;
	}
	public function setcapacity($capacity){
		$this->capacity = $capacity;
	}
	public function getcapacity(){
		return $this->capacity;
	}
	// destruktor
	public function __destruct(){
	}
}
?>